source/game/kart/KartParamFileManager.cc

```diff
@@ -64,8 +64,7 @@ EGG::RamStream KartParamFileManager::getDriverStream(Character character) const
     auto *file = reinterpret_cast<ParamFile<KartParam::Stats> *>(m_driverParam.file);
     ASSERT(file);
     void *offset = &file->params[idx];
-    u32 size = sizeof(KartParam::Stats);
-    return EGG::RamStream(reinterpret_cast<u8 *>(offset), size);
+    return EGG::RamStream(offset, sizeof(KartParam::Stats));
```

Comment:
This can be improved further by sending in `&file->params[idx]` instead of `offset`.